docs/en/tutorials/getting-started.rst

```diff
@@ -228,33 +227,196 @@ entity definition:
          * @var string
          */
         protected $name;
+    }
 
-        public function getId()
+When creating entity classes, all of the fields should be protected or private (not public).
+
+Adding the Entity behaviors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You have two options to define to define methods in your entity:
+**getters/setters**, or **mutators and DTOs**,
+respectively for **anemic models** or **rich models**.
+
+**Anemic models: Getters and setters**
+
+The most popular method is to create two kinds of methods to
+**read** (getter) and **update** (setter) the object's properties.
+
+Some fields such as ``$id`` are unlikely to be changed, so it's ok to
+omit them.
+
+.. note::
+
+    Setters are unrelated to Doctrine itself, because the ORM does not
+    use them to change the values of the fields. It instead relies on
+    the Reflection API and does not run either the constructor or other
+    methods.
+
+This method is mostly used when you want to focus on behavior-less
+entities, and when you need to move all your business logic in your
+services rather than in the objects themselves.
+
+It is a common convention which makes it possible to expose each field
+of your entity to external services, while allowing you to keep type
```

Comment:
s/external services/the external world

Reply:
s/keep type safety/keep some type safety

You said it yourself a bit further: invalid state ;-)